src/SourceLocator/ComposerSourceLocator.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace BetterReflection\SourceLocator;
+
+use Composer\Autoload\ClassLoader;
```

Comment:
If you import this class, you are also to require `composer/composer` in the project, no?